src/commands/format.rs

```diff
@@ -140,6 +147,15 @@ fn format_document(
     let formatter = Formatter::new(config);
     let formatted = formatter.format(&document)?;
 
+    if check {
+        if formatted != source {
+            print!("{}", StrComparison::new(&source, &formatted));
+            return Ok(1);
+        }
+        println!("ok");
```

Comment:
Can you update this print statement to include the path of what's "ok"? `{path} is formatted correctly`